base_layer/wallet/src/output_manager_service/service.rs

```diff
@@ -450,58 +386,59 @@ pub enum UTXOSelectionStrategy {
     Smallest,
 }
 
-#[cfg(test)]
-mod test {
-    use crate::output_manager_service::service::{OutputManagerService, PendingTransactionOutputs};
-    use chrono::Utc;
-    use rand::{CryptoRng, Rng, RngCore};
-    use tari_core::{
-        tari_amount::MicroTari,
-        transaction::UnblindedOutput,
-        types::{PrivateKey, PublicKey},
-    };
-    use tari_crypto::keys::{PublicKey as PublicKeyTrait, SecretKey};
-    use tari_service_framework::reply_channel;
-
-    fn make_output<R: Rng + CryptoRng>(rng: &mut R, val: MicroTari) -> UnblindedOutput {
-        let key = PrivateKey::random(rng);
-        UnblindedOutput::new(val, key, None)
-    }
-
-    #[test]
-    fn test_contains_output_function() {
-        let mut rng = rand::OsRng::new().unwrap();
-        let (secret_key, _public_key) = PublicKey::random_keypair(&mut rng);
-        let (_sender, receiver) = reply_channel::unbounded();
-        let mut oms = OutputManagerService::new(receiver, secret_key, "".to_string(), 0);
-        let mut balance = MicroTari::from(0);
-        for _i in 0..3 {
-            let uo = make_output(&mut rng.clone(), MicroTari::from(100 + rng.next_u64() % 1000));
-            balance += uo.value.clone();
-            oms.add_output(uo).unwrap();
-        }
-
-        let uo1 = make_output(&mut rng.clone(), MicroTari::from(100 + rng.next_u64() % 1000));
-
-        assert!(!oms.contains_output(&uo1));
-        oms.add_output(uo1.clone()).unwrap();
-        assert!(oms.contains_output(&uo1));
-
-        let uo2 = make_output(&mut rng.clone(), MicroTari::from(100 + rng.next_u64() % 1000));
-        assert!(!oms.contains_output(&uo2));
-        oms.spent_outputs.push(uo2.clone());
-        assert!(oms.contains_output(&uo2));
-
-        let uo3 = make_output(&mut rng.clone(), MicroTari::from(100 + rng.next_u64() % 1000));
-        assert!(!oms.contains_output(&uo3));
-        oms.pending_transactions.insert(1, PendingTransactionOutputs {
-            tx_id: 1,
-            outputs_to_be_received: vec![uo3.clone()],
-            outputs_to_be_spent: Vec::new(),
-            timestamp: Utc::now().naive_utc(),
-        });
-        assert!(oms.contains_output(&uo3));
-
-        assert_eq!(uo1.value + balance, oms.get_balance());
-    }
-}
+//#[cfg(test)]
```

Comment:
Can you update or delete these tests?

Reply:
Woops forogt those, was going to update them